manifests/mod/setenvif.pp

```diff
@@ -11,7 +11,7 @@
     ensure  => file,
     path    => "${apache::mod_dir}/setenvif.conf",
     mode    => $apache::file_mode,
-    content => template('apache/mod/setenvif.conf.erb'),
+    content => epp('apache/mod/setenvif.conf.epp'),
```

Comment:
this template doesn't have any params, is it expected?

Reply:
Yes, it doesnt have any params. This is expected behaviour